README.md

```diff
@@ -1,72 +1,40 @@
-# Auth0.swift
+![Auth0.swift](https://cdn.auth0.com/website/sdks/banners/swift-banner.png)
 
 [![CircleCI](https://img.shields.io/circleci/project/github/auth0/Auth0.swift.svg?style=flat)](https://circleci.com/gh/auth0/Auth0.swift/tree/master)
 ![Version](https://img.shields.io/cocoapods/v/Auth0.svg?style=flat)
 [![Coverage Status](https://img.shields.io/codecov/c/github/auth0/Auth0.swift/master.svg?style=flat)](https://codecov.io/github/auth0/Auth0.swift)
 ![License](https://img.shields.io/github/license/Auth0/Auth0.swift.svg?style=flat)
 
-Easily integrate Auth0 into iOS, macOS, tvOS, and watchOS apps. Add **login** and **logout**, store **credentials** securely, and access **user information**.
+📚 [**Documentation**](#documentation) **·** 🚀 [**Getting Started**](#getting-started) **·** 💻 [**API Reference**](#api-reference) **·** ❓ [**FAQs**](FAQ.md) **·** 💬 [**Feedback**](#feedback)
```

Comment:
To keep the links standard across the repos maybe just have the `FAQs` linked in the `Documentation` section?

Reply:
👍🏼 Just to confirm, is this an earlier design?

<img width="750" alt="Screen Shot 2022-10-21 at 07 23 38" src="https://user-images.githubusercontent.com/5055789/197174431-deea2fdc-fdab-42fd-abdf-955541193ac0.png">

Reply:
I think that was the first iteration yeah. I've been referencing the design titled `new banner` in the figma doc (I can't figure out how to link so partial screenshot below), I should have clarified on the slack channel sorry!

![image](https://user-images.githubusercontent.com/8705251/197178385-7bae76b9-c15f-4076-8de4-9235cde840f7.png)

Reply:
Since the links in this part are meant to reflect the README's H2 headers, should I remove the API docs one and include "Common Tasks" and "Support Policy"?

Reply:
Good point! Do you think the support policy could maybe be an item in the FAQs? And then maybe the common tasks could be maybe be covered by the `For other comprehensive examples see the EXAMPLES.md document`? Or do you think it's best to have those in the main readme?

Reply:
And the "Common Tasks" is meant as a Next Steps kind of thing right after the login, but to help guide and make sense of the raw list of features in EXAMPLES.md, so I think it's best placed in the README, right after Getting Started.
I mean, they're in the redesigned Flutter README as well.

Reply:
I think that's fair, I'd like to make sure Kevin and Stacy are cool with the additional link though so will ping them in the channel

Reply:
There's an analogous link in the Auth0.Android one (PR still open): https://github.com/auth0/Auth0.Android/blob/905d030cccde761904ff3a77c0a96460a51c9416/README.md?plain=1#L9

Reply:
Also in the JWTDecode.Android one: https://github.com/auth0/JWTDecode.Android/blob/c92149792709cb623c31dc9151e0ccf3a584fcb2/README.md?plain=1#L8